File: PS/Programmers/최대공약수와 최소공배수.cc
#include <string>
#include <vector>

using namespace std;

vector<int> solution(int a, int b) {
	vector<int> answer;
	int tmp,n,m;
	m = b;
	if (a < b)
	{
		tmp = a;
		a = b;
		b = tmp;
	}
	while (b!=0)
	{
		n = a % b;
		a = b;
		b = n;
	}
	answer.push_back(a);
	answer.push_back(tmp / a * m);
	return answer;
}

File: PS/Programmers/최대공약수와 최소공배수.py
def solution(n, m):
    answer = []
    g = gcd(n,m)
    answer.append(g)
    answer.append(int(n*m/g))
    return answer
def gcd(a,b):
    if a>b:
        a,b=b,a
    while(b!=0):
        n=a%b
        a=b
        b=n
    return a
    